code/__HELPERS/unsorted.dm

```diff
@@ -1124,32 +1124,20 @@ Returns 1 if the chain up to the area contains the given typepath
 		loc = loc.loc
 	return null
 
-/proc/get_turf_or_move(turf/location)
-	return get_turf(location)
-
-
 //For objects that should embed, but make no sense being is_sharp or is_pointed()
 //e.g: rods
 GLOBAL_LIST_INIT(can_embed_types, typecacheof(list(
 	/obj/item/stack/rods,
 	/obj/item/pipe)))
 
 /proc/can_embed(obj/item/W)
-	if(is_sharp(W))
-		return 1
+	if(W.sharp)
+		return TRUE
 	if(is_pointed(W))
-		return 1
+		return TRUE
 
 	if(is_type_in_typecache(W, GLOB.can_embed_types))
-		return 1
-
-//Whether or not the given item counts as sharp in terms of dealing damage
-/proc/is_sharp(obj/O)
-	if(!O)
```

Comment:
Be sure that the null check here is either also done, or isn't needed

Reply:
After looking into it, pretty much every use was in some place that if it was done with no item it'd runtime, like `attackby`. The only place I see it maybe having issues is `adjustBruteLossByPart` so I'll test if that works, if not, I'll add a nullcheck